index.html

```diff
@@ -19,9 +19,9 @@
 
 <body>
     <p>
-        <h1 id="name"><b>San Diego Top Spots</b></h1>
+        <h1 id="name">San Diego Top Spots</h1>
         <p>
-            There are some great places to see in downtown San Diego. In this project you'll get to know a few of them by turning the provided JSON file into a table in HTML with links to Google Maps using jQuery.
+            <h4>There are some great places to see in downtown San Diego. In this project you'll get to know a few of them by turning the provided JSON file into a table in HTML with links to Google Maps using jQuery.</h4>
             <p>
```

Comment:
Find a better way to style this than using p tags, if needed add a div and style according to what you wanted.